### PR TITLE
WIP feat(MOPLAT-255): Default to production in release builds / staging in builds

### PR DESCRIPTION
**Pull request by patrinoua**
### Description

This PR resolves [MOPLAT-255]

https://stackoverflow.com/questions/55794280/jest-fails-with-unexpected-token-on-import-statement

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-255]: https://artsyproduct.atlassian.net/browse/MOPLAT-255?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/store/Models/EnvironmentModel.ts [hunk 1]
```diff
@@ -34,19 +35,30 @@ export const environmentOptions = defineEnvironmentOptions({
 export type EnvironmentKey = keyof typeof environmentOptions
 
 export interface EnvironmentModel {
+  isTestFlight: boolean
   activeEnvironment: Environment
   strings: Computed<this, { [k in EnvironmentKey]: string }>
   setEnvironment: Action<this, Environment>
 }
 
+console.log("hello from EnvironmentModel. RNTestFlight.isTestFlight?", RNTestFlight.isTestFlight)
```
Comment (patrinoua) [inline src/app/store/Models/EnvironmentModel.ts:44]:
```suggestion
```

File: src/app/store/Models/EnvironmentModel.ts [hunk 2]
```diff
@@ -34,19 +35,30 @@ export const environmentOptions = defineEnvironmentOptions({
...
+
+if (RNTestFlight.isTestFlight) {
+  console.log("Ground control to Major Tom!!!")
+}
+
+console.log("==== EnvironmentModel: activeEnvironment", RNTestFlight.isTestFlight)
+console.log("==== EnvironmentModel: __DEV__", __DEV__)
```
Comment (patrinoua) [inline src/app/store/Models/EnvironmentModel.ts:51]:
```suggestion

```

File: src/app/screens/HomeTabs/HomeTabs.tsx [hunk 3]
```diff
@@ -16,40 +16,16 @@ export const HomeTabs = () => {
   const navigation = useNavigation<NavigationProp<HomeTabsScreens>>()
   const selectedPartner = GlobalStore.useAppState((state) => state.activePartnerID)
   const { tabName } = useRoute<HomeTabsRoute>().params || { tabName: "Artists" }
-
+  const environment = GlobalStore.useAppState((state) => state.config.environment)
+  const activeEnvironment = environment.activeEnvironment
+  const isTestFlight = environment.isTestFlight
+  console.log(">>>>> HomeTabs: activeEnvironment, isTestFlight", activeEnvironment, isTestFlight)
```
Comment (patrinoua) [inline src/app/screens/HomeTabs/HomeTabs.tsx:22]:
```suggestion

```

**Comment by brainbicycle** [comment]
closing old prs, let me know if anything still needs addressing here!

**Pull request closed without merging**